packages/react-core/src/components/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -21,27 +21,41 @@ describe('TextArea', () => {
     expect(props.onChange).toHaveBeenCalledWith('a', expect.any(Object));
   });
 
-  test('simple text input', () => {
+  test('simple text area', () => {
```

Comment:
nit - just an inconsistency, I see the use of `textarea` (the html element), text area (more of a description, also the component title in our docs), and TextArea in the doc and test comments. I don't know if it's worth attempting to normalize that usage.

Same with the text input docs. In the component prop docs, it's "TextInput" and just "input".

Also looks like we use "readonly" and "read only" in the same context - when it's a string, describing that something is `readonly`

```suggestion
  test('simple text area', () => {
```

Reply:
Thank you @mcoker, nice catch! The standard is to use sentence case in our prop docs, so I've updated the `TextArea` & `TextInput` docs to reflect that. 

Also standardizing on the string "read only" since it looks like the tests were initially written using that verbiage.